src/doi_finder.py

```diff
@@ -89,7 +91,8 @@ def crossref_auth_title_to_doi(author, title):
     # use only surname of first author
     browser["auth2"] = re.sub(r'[A-Z] ', ' ',
         re.sub(r'[^a-zA-Z0-9 ]+', ' ', author).split("and")[0])
-    browser["atitle2"] = re.sub(r'[^a-zA-Z0-9 ]+', ' ', title)
+    #browser["atitle2"] = re.sub(r'[^a-zA-Z0-9 ]+', ' ', title)
+    browser["atitle2"] = re.sub(r'[^a-zA-Z0-9,:. ]+', ' ', title)
```

Comment:
hmm, I don't know it any more, but I probably had a reason to match only the first part of the name, before any commas.